lib/rsgem/ci_providers/base.rb

```diff
@@ -13,6 +13,7 @@ def initialize(config_file_source: nil, config_file_destination: nil, display_na
       end
 
       def install(context)
+        remove_travis(context)
```

Comment:
is this GH Actions specific? Isn't it going to delete it also when Travis is the selected CI provider?

Reply:
That's true we always remove it, because we don't really need it. 

We just copy over our own travis.yml file.